README.md

```diff
@@ -93,13 +93,13 @@ The fix is to drop the existing database and to restart the service.  In docker-
 can be done by running
 
 ```bash
-docker-compose run backend psql postgres -c 'drop database nmdc;'
+docker-compose run backend psql postgres -c 'drop database nmdc_a;'
```

Comment:
Locally, I've had to also set my alembic.ini database uri to `nmdc_a` rather than `nmdc` -- should we also make that change in the repo?

Reply:
Yes.